lib/node_modules/@stdlib/math/base/special/gcdf/README.md

```diff
@@ -0,0 +1,243 @@
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# gcdf
+
+> Compute the [greatest common divisor][gcd] (gcd).
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+The [greatest common divisor][gcd] (gcd) of two non-zero integers `a` and `b` is the largest positive integer which divides both `a` and `b` without a remainder. The gcd is also known as the **greatest common factor** (gcf), **highest common factor** (hcf), **highest common divisor**, and **greatest common measure** (gcm).
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var gcdf = require( '@stdlib/math/base/special/gcdf' );
+```
+
+#### gcdf( a, b )
+
+Computes the [greatest common divisor][gcd] (gcd).
+
+```javascript
+var v = gcdf( 48, 18 );
+// returns 6
+```
+
+If both `a` and `b` are `0`, the function returns `0`.
+
+```javascript
+var v = gcdf( 0, 0 );
+// returns 0
+```
+
+Both `a` and `b` must have integer values; otherwise, the function returns `NaN`.
+
+```javascript
+var v = gcdf( 3.14, 18 );
+// returns NaN
+
+v = gcdf( 48, 3.14 );
+// returns NaN
+
+v = gcdf( NaN, 18 );
+// returns NaN
+
+v = gcdf( 48, NaN );
+// returns NaN
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var gcdf = require( '@stdlib/math/base/special/gcdf' );
+
+var a = discreteUniform( 100, 0, 50 );
+var b = discreteUniform( a.length, 0, 50 );
+
+var i;
+for ( i = 0; i < a.length; i++ ) {
+    console.log( 'gcd(%d,%d) = %d', a[ i ], b[ i ], gcdf( a[ i ], b[ i ] ) );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/gcdf.h"
+```
+
+#### stdlib_base_gcdf( a, b )
+
+Computes the greatest common divisor (gcd).
+
+```c
+float v = stdlib_base_gcdf( 48.0, 18.0 );
+// returns 6.0
```

Comment:
```suggestion
float v = stdlib_base_gcdf( 48.0f, 18.0f );
// returns 6.0f
```

We should have a suffix `f` when dealing with single-precision floating-point numbers in C.